modules/ptp-configuring-linuxptp-services-as-ha-bc-for-dual-nic.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ptp-configuring-linuxptp-services-as-ha-bc-for-dual-nic_{context}"]
+= Configuring linuxptp as highly available boundary clock for dual NIC Intel e810 hardware
+
+You can configure `linuxptp` services (`ptp4l`, `phc2sys`) as a highly available boundary clock (T-BC) for dual Intel e810 Salem channel dual NIC hardware.
+You connect each NIC to the same upstream leader clock with separate `ptp4l` instances for each NIC feeding the downstream clocks.
+
+Create two `PtpConfig` custom resource (CR) objects for each NIC and a third `PtpConfig` CR that configures high availability between the two NICs.
+
+The third `ptpConfig` CR configures the `ptp4l` and `phc2sys` services for network clock synchronization.
+The third CR adds profiles for the `ptp4l` configurations under the `spec.profile.ptpSettings.haProfiles` key and passes the kernel socket path of those profiles to the `phc2sys` service.
```

Comment:
HA Profile configuration aims to establish a Highly Available (HA) SystemClock service using Dual nic Boundary Clocks.
Boundary Clocks: Two Boundary Clock instances will participate in the HA setup, each with its own configuration profile.